### PR TITLE
about us page created

### PR DESCRIPTION
**Pull request by dkasb**

### CODE REVIEW

File: includes/Login.php [hunk 1]
```diff
@@ -58,17 +63,20 @@
...
+<table style ="border-width:5px; border-style:double ;border-color:white ;padding: 8px;">
 
 <caption style ="color:white;text-align:center"><h1><b>WELCOME</b></h1></CAPTION>
 <tr><th style = "background-color:#b3003b ;color:white">Username  :</th><td> <input type="text" name="User" required autofocus/> </td></tr>
 <tr><th style = "background-color:#b3003b ; color:white">Password  : </th><td><input type="password" name="Passwd" required/> </td></tr>
 
 </table>
 <br><br><br><br>
-<input  type="submit" name="login" value="Login!" style="cursor: pointer; font-size : 30px; height:auto; width:auto ;background-color: 	#000080 ;border:15px ;color:white" />
+<input  type="submit" name="login" value="Login!" style="cursor: pointer; font-size : 30px; height:auto; width:auto ;background-color: 	transparent ;border:1px solid white;color:white" />
```
Comment (pkgodara) [inline includes/Login.php:74]: __[code_sniffer]__
* Line exceeds 120 characters; contains 187 characters

File: includes/AboutUs.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          
+         <center>    <h2><i><b> ...Go Paperless, Go Green </b></i></h2></center>
+           
+           
+          </div>
+        </div>
+        <div class="w3-container">
+       <hr><hr> <h3>Make Offices Digital and Efficient....</h3>
+            <hr><hr>
+             <img class="w3-round w3-margin-right" src="../image/home.png" style="width:25%;"><span class="w3-opacity w3-large"> IIT PATNA</span>
```
Comment (pkgodara) [inline includes/AboutUs.php:54]: __[code_sniffer]__
* Line exceeds 120 characters; contains 145 characters

File: includes/AboutUs.php [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            <hr><hr>
+              <img class="w3-round w3-margin-right" src="../image/email.png" style="width:25%;"><span class="w3-opacity w3-large"><h5><ul><li>
```
Comment (pkgodara) [inline includes/AboutUs.php:56]: __[code_sniffer]__
* Line exceeds 120 characters; contains 153 characters

File: includes/AboutUs.php [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      </div><br><hr>
+
+    <!-- End Left Column -->
+    </div>
+
+    <!-- Right Column -->
+    <div class="w3-twothird">
+    
+      <div class="w3-container w3-card-2 w3-white w3-margin-bottom">
+        <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-suitcase fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>About eOrg</h2>
```
Comment (pkgodara) [inline includes/AboutUs.php:75]: __[code_sniffer]__
* Line exceeds 120 characters; contains 138 characters

File: includes/AboutUs.php [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        <div class="w3-container">
+       <h4><i> <center> eOrg is a free open source software intended to reduce loads of paperwork<br> done inter-organisational office-work.</center></i></h4>
```
Comment (pkgodara) [inline includes/AboutUs.php:77]: __[code_sniffer]__
* Line exceeds 120 characters; contains 158 characters

File: includes/AboutUs.php [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+          <hr>
+        </div>
+        <div class="w3-container">
+           <h1 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>Our Intensions :</h1>
```
Comment (pkgodara) [inline includes/AboutUs.php:81]: __[code_sniffer]__
* Line exceeds 120 characters; contains 150 characters

File: includes/AboutUs.php [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+	<li>To build an enhanced structure for better application management within organisation.
+
+	<li>Proficient and secure way to manage different types of documents/applications within an organisation.
+</ul>
+</h3>
+          <hr>
+        </div>
+        <div class="w3-container">
+       <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>License</h2>
```
Comment (pkgodara) [inline includes/AboutUs.php:92]: __[code_sniffer]__
* Line exceeds 120 characters; contains 137 characters

File: includes/AboutUs.php [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    eOrg is distributed under the <a href = "https://opensource.org/licenses/MIT">MIT License.</a><br><br>
+      </div>
+      </div>
+
+      <div class="w3-container w3-card-2 w3-white">
+        <div class="w3-container">
+           <img class="w3-round w3-margin-right" src="../image/aboutus.png" style="width:25%;"><span class="w3-opacity w3-large"></span>
```
Comment (pkgodara) [inline includes/AboutUs.php:99]: __[code_sniffer]__
* Line exceeds 120 characters; contains 136 characters

File: includes/AboutUs.php [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+        NABEEL QAISER ,   CSE STUDENT AT IIT PATNA
+      <li>
+        PRADEEP KUMAR ,   CSE STUDENT AT IIT PATNA
+       <li>
+	DEEPAK VERMA  ,   CSE STUDENT AT IIT PATNA
+        </ul>
+        </div> <hr>  </div>
+        <div class="w3-container">
+      
+ <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>Want To Contribute...</h2>
```
Comment (pkgodara) [inline includes/AboutUs.php:110]: __[code_sniffer]__
* Line exceeds 120 characters; contains 145 characters

File: includes/AboutUs.php [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+</html>
+
+HTML;
+echo $html;
+
+
+
+
+
+?>
```
Comment (pkgodara) [inline includes/AboutUs.php:157]: __[code_sniffer]__
* A closing tag is not permitted at the end of a PHP file

**Comment by pkgodara** [comment]
Click here if you want to create code_sniffer's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/80422762/issues/pull_request/58/diff/7b164fa61b03bb759174f5cd8cb0d3529f96911c?open_modal=1#code_sniffer)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-code_sniffer)

**Review by NabeelQaiser — APPROVED** [review]

**Pull request merged**